orchestra/static/orchestra/project/js/data_service.js

```diff
@@ -0,0 +1,90 @@
+var serviceModule =  angular.module('orchestra.project.services');
+
+serviceModule.factory('dataService', function() {
+  var dataService = new function() {
+    var that = this;
+    this._meta = {'tasks': {}};
```

Comment:
there might be a better name for this than meta, since it's the actual task data, no?